wiki/Difficulties/en.md

```diff
@@ -1,141 +1,59 @@
-Difficulty appropriate gameplay elements
-==========================================
+# Difficulties
 
-Star Rating and Web Icons
--------------------------
+*Not to be confused with: [Beatmaps](/wiki/Beatmaps).*
 
-![Star Rating Range](SR_range.png "Star Rating Range")
+*See also: [Ranking Criteria § Mapset](/wiki/Ranking_Criteria/#mapset).*
 
--   From the [Ranking Criteria](/wiki/Ranking_Criteria):
-    -   Below 1.5: Easy ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")
-    -   Below 2.25: Normal ![Normal icon](/wiki/shared/diff/normal-s.png "Normal icon")
-    -   Below 3.75: Hard ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")
-    -   Below 5.25: Insane ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon")
-    -   Above 5.25: Expert ![Expert icon](/wiki/shared/diff/expert-s.png "Expert icon")
+The difficulty of a beatmap *describes* the level of skill needed to complete a beatmap.
```

Comment:
The difficulty of a beatmap *describes* the level of skill needed to complete it.